salt/pillar/postgres.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+'''
+Retrieve Pillar data by doing a postgres query
+
+:maturity: new
+:depends: psycopg2
+:platform: all
+
+Complete example
+=====================================
+
+.. code-block:: yaml
+
+    postgres:
+      user: 'salt'
+      pass: 'super_secret_password'
+      db: 'salt_db'
```

Comment:
Right now, you don't have any support for all the extra [Parameter Key Words](https://www.postgresql.org/docs/9.2/static/libpq-connect.html) that are available via using the connection_uri.

But looking at all the other postgres stuff in salt, I am of the mind that we should accept this as is, and then go back and update all the postgres returners and queues to support the connection_uri.